ui/app/components/toolbar-button.js

```diff
@@ -0,0 +1,6 @@
+import LinkComponent from '@ember/routing/link-component';
+
+export default LinkComponent.extend({
+  classNames: ['toolbar-button'],
+  glyph: 'chevron-right',
```

Comment:
It doesn't look like we pass this in anywhere, could it be just `chevron-right` in the template?